common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -89,8 +89,16 @@ public void onContainerRender(ContainerRenderEvent event) {
             int y = containerScreen.topPos;
             switch (emeraldCountType.get()) {
                 case TEXT -> renderTextCount(event.getPoseStack(), textX, y, topEmeralds);
-                case TEXTURE -> renderTexturedCount(
-                        event.getPoseStack(), textureX, y, topEmeralds, showZerosInEmeraldCount.get());
+                case TEXTURE -> {
+                    int displayedTextures = (int) (Arrays.stream(getEmeraldAmounts(topEmeralds))
```

Comment:
Is this calculating the height of the textures that renderTexturedCount() is rendering? It feels a bit complex-y, and it is not immediately clear why we should calculate it like this. I get the feeling this is based on some other code elsewhere, but there is no connection to be found.